operational/stranded_servers/Stranded_Servers.pt

```diff
@@ -0,0 +1,191 @@
+name "Stranded Servers"
+rs_pt_ver 20180301
+type "policy"
+short_description "Report and remediate any Servers that are stranded in booting. \n See the [README](https://github.com/rightscale/policy_templates/tree/master/operational/stranded_servers) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
+long_description "Version: 1.0"
+category "Operational"
+severity "low"
+
+###############################################################################
+# Permissions
+###############################################################################
+
+permission "perm_read_creds" do
```

Comment:
I don't see any reference to cred() or use the credentials.  can this be removed?

Reply:
Made Changes.